pbsmetrics/go_metrics.go

```diff
@@ -280,9 +287,9 @@ func (me *Metrics) getAccountMetrics(id string) *accountMetrics {
 	am.bidsReceivedMeter = metrics.GetOrRegisterMeter(fmt.Sprintf("account.%s.bids_received", id), me.MetricsRegistry)
 	am.priceHistogram = metrics.GetOrRegisterHistogram(fmt.Sprintf("account.%s.prices", id), me.MetricsRegistry, metrics.NewExpDecaySample(1028, 0.015))
 	am.adapterMetrics = make(map[openrtb_ext.BidderName]*AdapterMetrics, len(me.exchanges))
-	for _, a := range me.exchanges {
-		am.adapterMetrics[a] = makeBlankAdapterMetrics()
-		registerAdapterMetrics(me.MetricsRegistry, fmt.Sprintf("account.%s", id), string(a), am.adapterMetrics[a])
+	for i := 0; i < len(me.exchanges) && me.MetricsDisabled.AccountAdapterDetails; i++ {
```

Comment:
not sure if putting the `me.MetricsDisabled.AccountAdapterDetails` check in an if outside the for would be marginally better, as you wouldn't have to recheck the value each loop. Then again this is startup code, so won't impact running performance.

Reply:
You are absolutely right. Wow I'm missing a test case then, let me double check them